ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -382,6 +382,18 @@ keys:
                 new_key: listen_ranges
             session_tracker:
               type: str
+            shared_secret:
+              type: dict
+              keys:
+                profile:
```

Comment:
make `profile` and `hash_algorithm` required keys

Reply:
Fixed.